rhel7/profiles/ospp42.profile

```diff
@@ -1,204 +0,0 @@
-documentation_complete: true
```

Comment:
The `ospp.profile` file should be removed instead. content of `ospp42.profile` file should be kept, because this one is more up2date, but  the file should be renamed to `ospp42.profile`.

Reply:
Please remove also the ospp42 from the stable profile IDs test.